ref/config/serverConfiguration.adoc

```diff
@@ -24,171 +18,134 @@ following example:
 
-The `server.env` file format is in terms of `KEY = value` and does not support
-variable substitution.  For example:
+. `${wlp.install.dir}/etc/`
+. `${wlp.user.dir}/shared/`
+. `${server.config.dir}/`
 
-    JAVA_HOME=/opt/ibm/java
-    WLP_USER_DIR=/opt/wlp-usr
-Key values must not contain white space. The values are interpreted as literal so there is no need to escape spaces or other special characters.
+If the same property is set in multiple locations, then the last value found is used.
 
-== jvm.options
-The `jvm.options` files are optional but, when present, are read by the
-`bin/server` shell script to determine what options to use when
-launching the JVM for {projectName}. The `jvm.options` files are read
-from the following locations in order:
+The most common use of these files is to set the following environment variables:
 
-* `${wlp.user.dir}/shared/jvm.options`
-* `${server.config.dir}/configDropins/defaults/`
-* `${server.config.dir}/`
-* `${server.config.dir}/configDropins/overrides/`
+* `JAVA_HOME`: Indicates which JVM to use. If this is not set, the system default is used.
+* `WLP_USER_DIR`: Indicates the location of the `usr` directory that contains the server config. This can only be set in the `etc/server.env` file because the other locations are relative to the `usr` directory.
+* `WLP_OUTPUT_DIR`: Indicates where the server writes files to. By default, the server writes to the directory structure that the config is read from. However, in some secure profiles the server config needs to be read-only so the server must write files to another location.
 
-If none of these files exist, the server script looks for a file in
-`${wlp.install.dir}/etc` and uses that.
+The `server.env` file is in `KEY=value` format, as shown in the following example:
+
+[source,properties]
+----
+JAVA_HOME=/opt/ibm/java
+WLP_USER_DIR=/opt/wlp-usr
+----
+
+Key values must not contain white space. The values are interpreted literally so you don't need escape spaces or other special characters. These files don't support variable substitution.
+
+[#jvm-options]
+=== `jvm.options`
+The `jvm.options` files are optional. These files are read by the `bin/server` shell script to determine what options to use when the JVM is launched for {projectName}. `jvm.options` files are read from the following locations in order:
+
+. `${wlp.user.dir}/shared/jvm.options`
+. `${server.config.dir}/configDropins/defaults/`
+. `${server.config.dir}/`
+. `${server.config.dir}/configDropins/overrides/`
+
+If no `jvm.options` files exist in these locations, then the server script looks for the file in `${wlp.install.dir}/etc`, if such a directory exists.
 
 Common uses of `jvm.options` files include:
 
 * Setting JVM memory limits
-* Enabling Java Agents provided by monitoring products
-* Setting Java System Properties
+* Enabling Java Agents that are provided by monitoring products
+* Setting Java system properties
```

Comment:
I think System Properties is likely correct since System refers to a class `java.lang.System` and the Properties are accessible via `getProperties`

Reply:
Updated in both places